GDJS/tests/benchmarks/layer.js

```diff
@@ -10,7 +10,8 @@ describe('gdjs.Layer', function() {
   var runtimeScene = new gdjs.RuntimeScene(runtimeGame);
 
   it('benchmark convertCoords and convertInverseCoords', function() {
-    this.timeout(20000);
+    // TODO EBO Understand why the timeout need to be 35s instead of 20s to pass on CI.
+    this.timeout(35000);
```

Comment:
This is not great but I guess it because of the call depth that comes with generalization.

Reply:
Resolving as fixed.